webapp/src/client.ts

```diff
@@ -692,6 +693,14 @@ export const saveChannelAction = async (action: ChannelAction): Promise<string>
     return action.id;
 };
 
+export const updateRunActions = async (playbookRunID: string, actions: RunActions) => {
+    try {
+        return await doPut(`${apiUrl}/runs/${playbookRunID}/actions`, JSON.stringify(actions));
+    } catch (error) {
+        return {error};
```

Comment:
slightly more accurate errors (will be forced in typescript 4.4)
```suggestion
        return {error as ClientError};
```

Reply:
to give some context, we are still in 4.3, and errors from the catch are type=any, in 4.4 they become type=unknown so you can not use properties unless you cast the error.

https://dev.to/ankittanna/what-s-new-in-typescript-4-4-ai7

Reply:
This cannot be casted directly to `ClientError`. I propose to leave it as is and come back to this later, I think it'd be good to talk about how to handle errors in the client.